doc/md/monitoring/03-overview.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: How it works
+---
+
+## Overview
+
+This document provides an overview of how Atlas Schema Monitoring works. It describes the high level architecture,
+key capabilities and features.
+
+### Architecture
+
+![](https://atlasgo.io/uploads/cloud/monitoring/arch.png)
+
+Atlas never has direct access to your database, instead it uses a middleman, the Atlas agent, to connect to your
+database instead. In order for this to work, the agent needs to be installed in the same VPC as the database to obtain
```

Comment:
Installed somewhere with network connectivity to the database, typically within ..